.github/workflows/icfy-stats.yml

```diff
@@ -21,12 +21,6 @@ jobs:
         env:
           PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: true
         run: yarn install
```

Comment:
`yarn install --inline-builds` will give us a more verbose output in case of error.

Reply:
Good idea!